bucket/milkman.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "5.1.0",
+    "description": "An Extensible Request/Response Workbench",
+    "homepage": "https://github.com/warmuuh/milkman",
+    "license": "MIT",
+    "url": [
+        "https://github.com/warmuuh/milkman/releases/download/5.1.0/milkman-dist-win64-bin.zip",
+        "https://github.com/warmuuh/milkman/releases/download/5.1.0/milkman-dist-plugins-bin.zip"
+    ],
+    "hash": [
+        "a332f698efe992f9da49fa36b9b96f3bd03e3a4f8238056ef486703a16e5d45b",
+        "83d1db252d35de8f2a987e55a9ad01e4f23706b728b1d8188b59b2f362defec3"
+    ],
```

Comment:
64bit only

Reply:
@Ash258 the second one is plugins, not 32bit application.

Reply:
So the users will be able to run random plugins jars, but not the main application because it will not work on their platform?? Seems like good idea.

Reply:
@Ash258 Milkman only have 64bit binaries. If scoop does not accept 64bit only applications, I'll close this PR.

Reply:
https://github.com/lukesampson/scoop/blob/master/schema.json